src/DependencyInjection/Configuration.php

```diff
@@ -768,6 +769,12 @@ private function createCachePluginNode()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('cache_listeners')
```

Comment:
at the top of this configuration tree, below the line `fixXmlConfig('respect_response_cache_directive')` we need a fixXmlConfig for 'cache_listeners'.

Reply:
done.